variation/schemas/schemas.py

```diff
@@ -0,0 +1,17 @@
+"""Module for general schemas used throughout the app"""
+
+
+from enum import Enum
+
+
+class Endpoint(str, Enum):
+    """Define contrains for endpoint names"""
+
+    TO_VRS = "toVRS"
```

Comment:
any reason why this isn't snake case?

Reply:
I was just going with the actual endpoints that we use